docs/examples/tutorial/clibraries/queue3.py

```diff
@@ -0,0 +1,67 @@
+from cython.cimports import cqueue
```

Comment:
`import cython` is worth having here.
In fact, given the long `cast()` lines above, I'd even additionally add `from cython import cast` and use that below.

Reply:
done.